server.js

```diff
@@ -0,0 +1,31 @@
+const express = require("express");
```

Comment:
## Critical

I totally forgot this but since we are transpiling the `server.js` down to ES5 then let's used `ES6` styled imports.

So let's have something like:
```
import express from 'express';
```

instead of what we have there.